webfront/views/modifiers.py

```diff
@@ -915,6 +916,25 @@ def show_subset(value, general_handler):
     general_handler.queryset_manager.show_subset = True
 
 
+def get_interpro_n_matches(value, general_handler):
+
+    queryset = general_handler.queryset_manager.get_queryset().first()
+    ipro_n_matches = InterProNMatches.objects.filter(protein_acc=queryset.accession)
+    ipro_n_result = {}
+
+    for match in ipro_n_matches: 
+        ipro_n_result[match.entry.pk] = {
+            "accession": match.entry.pk,
+            "name": match.entry.name,
+            "type": match.entry.type, 
+            "short_name": match.entry.short_name,
+            "source_database": match.entry.source_database,
+            "locations": loads(match.locations)
```

Comment:
The InterPro entry in which the match's signature might be integrated should be reported as well.